src/e2e/java/teammates/e2e/cases/lnp/StudentProfileLNPTest.java

```diff
@@ -108,29 +104,9 @@ protected LNPTestData getTestData() {
                 return students;
             }
 
-            @Override
-            protected Map<String, FeedbackSessionAttributes> generateFeedbackSessions() {
-                return new HashMap<>();
-            }
-
-            @Override
-            protected Map<String, FeedbackQuestionAttributes> generateFeedbackQuestions() {
-                return new HashMap<>();
-            }
-
-            @Override
-            protected Map<String, FeedbackResponseAttributes> generateFeedbackResponses() {
-                return new HashMap<>();
-            }
-
-            @Override
-            protected Map<String, FeedbackResponseCommentAttributes> generateFeedbackResponseComments() {
-                return new HashMap<>();
-            }
-
             @Override
             protected Map<String, StudentProfileAttributes> generateProfiles() {
-                Map<String, StudentProfileAttributes> profiles = new HashMap<>();
+                Map<String, StudentProfileAttributes> profiles = new LinkedHashMap<>();
```

Comment:
Sorry, can you clarify why you changed this from `HashMap` to `LinkedHashMap`

Reply:
The created data is stored in the order of creation (numerical order) in the json and csv files. Pretty minor, but is more intuitive to us humans when we are debugging with the data